dbms/src/Operators/BlockInputStreamSourceOp.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <Common/Logger.h>
+#include <Operators/Operator.h>
+
+namespace DB
+{
+class IBlockInputStream;
+using BlockInputStreamPtr = std::shared_ptr<IBlockInputStream>;
+
+// wrap for BlockInputStream
```

Comment:
and used where?

Reply:
So maybe comment it?

Reply:
I'm not so sure if it will be used by other classes in the future

Reply:
But now I can add this comment.

Reply:
comment added.